src/content/get-started/install/chromeos/android.md

```diff
@@ -10,12 +10,12 @@ next:
   path: /get-started/test-drive
 ---
 
-{% include docs/install/reqs/linux/base.md os=devos target=target -%}
+{% include docs/install/reqs/linux/base.md os=devos target=target %}
```

Comment:
What's this change doing? Is there a special syntax we need to use for includes?

Reply:
This is a [LiquidJS templating language](https://liquidjs.com/) setting. Including the `-` tells the rendering engine to remove any whitespace "next to" it in Markdown before rendering it as HTML.

Example:

```markdown
Text 1

{%- render docs/foo/bar.md %}

Text 2
```

This will remove any whitespace, including line breaks, between `Text 1` and the include. That's what this PR does. It removes all of those "remove whitespace" indicators for all install pages. That will fix the issue raised.

Reply:
Ah, that's super helpful, thanks. Maybe this should go into some sort of "best practices" for working with includes in the website.

Reply:
Yeah, there needs to be a greater explanation of the feature set. I've had to do a ton of research.

Can this get an approval now? Thanks!